tasks/managed/add-fbc-contribution/add-fbc-contribution.yaml

```diff
@@ -133,69 +132,100 @@ spec:
         echo -n "$timestamp" > "$(results.buildTimestamp.path)"
         echo -n "$target_index" > "$(results.requestTargetIndex.path)"
```

Comment:
What about this one? Even with multiple components, it's still just a single value? So the idea is that you can add several fragments at once, but all of them for the same target index image?

Reply:
The `targetIndex` is indeed always the same. The timestamp itself is used for pre-ga targetIndexes, which should be unique as well.  It was a nice find btw, because this need to be out of the loop.

Reply:
Nah, it is already out of the loop, so great. That is correct.